zk_toolbox/crates/zk_inception/src/utils/consensus.rs

```diff
@@ -0,0 +1,88 @@
+use config::ChainConfig;
+use secrecy::{ExposeSecret, Secret};
+use zksync_config::configs::consensus::{
+    AttesterPublicKey, AttesterSecretKey, ConsensusSecrets, GenesisSpec, NodePublicKey,
+    NodeSecretKey, ProtocolVersion, ValidatorPublicKey, ValidatorSecretKey, WeightedAttester,
+    WeightedValidator,
+};
+use zksync_consensus_crypto::{Text, TextFmt};
+use zksync_consensus_roles as roles;
+
+struct ConsensusKeys {
+    validator_key: roles::validator::SecretKey,
+    attester_key: roles::attester::SecretKey,
+    node_key: roles::node::SecretKey,
+}
+
+struct ConsensusPublicKeys {
+    validator_key: roles::validator::PublicKey,
+    attester_key: roles::attester::PublicKey,
+}
+
+pub fn generate_consensus_keys() -> ConsensusKeys {
+    ConsensusKeys {
+        validator_key: roles::validator::SecretKey::generate(),
+        attester_key: roles::attester::SecretKey::generate(),
+        node_key: roles::node::SecretKey::generate(),
+    }
+}
+
+fn get_consensus_public_keys(consensus_keys: &ConsensusKeys) -> ConsensusPublicKeys {
+    ConsensusPublicKeys {
+        validator_key: consensus_keys.validator_key.public(),
+        attester_key: consensus_keys.attester_key.public(),
+    }
```

Comment:
what happened to the node_key?

Reply:
It is not needed:
https://github.com/matter-labs/zksync-era/blob/matias-zki-consensus/zk_toolbox/crates/zk_inception/src/utils/consensus.rs#L82